www/assets/css/account.css

```diff
@@ -293,6 +293,7 @@ body.theme-b {
     left: 50%;
     transform: translate(-50%, -50%);
     max-width: 800px;
+    z-index: 10000001;
```

Comment:
only for media query?

Reply:
Yes, it seems only necessary for wide screens

Reply:
![image](https://github.com/catchpoint/WebPageTest/assets/51945810/e28598e9-54e6-414f-9c18-fb9acf33c470)